openreview/matching.py

```diff
@@ -80,3 +82,294 @@ def get_domains(entity, subdomains = False):
         return valid_domains
     else:
         return full_domain
+
+def match(client, configuration_note, Solver):
```

Comment:
I think this piece of code is too important to have it as a helper of the library, why don't we add it to the matching code?

We can keep a part that is agnostic to openreview, but we will need a layer to integrate both parts and [this](https://github.com/iesl/openreview-matching) repo should be the place.